Makefile.inc

```diff
@@ -94,3 +94,16 @@ endif
 
 CODEOWNERS: CODEOWNERS.in
 	$(SCRIPTS_DIR)/gen-codeowners
+
+# This removes all Submariner-provided images and all untagged images
+# Use this to ensure you use current images
+# Copied from Makefile to provide this everywhere (until we can share
+# non-Dapper goals across projects)
+prune-subm-images:
```

Comment:
Won't this try to remove the temp image used by `make shell` and fail?

Reply:
Yes, but we’re not running with `set -e` so the other prunable images are still pruned.